src/pages/register-a-beacon/vessel-communications.tsx

```diff
@@ -69,12 +63,15 @@ const VesselCommunications: FunctionComponent<DraftBeaconUsePageProps> = ({
       pageHeading={pageHeading}
       showCookieBanner={showCookieBanner}
       formErrors={form.errorSummary}
-      pageText={pageText}
-      headingType="legend"
     >
+      <FormFieldsetAndLegend
+        pageHeading={pageHeading}
```

Comment:
Page heading should have (Optional) in as all the checkboxes are optional.

See: https://trello.com/1/cards/6108eb640d25af412f8264f7/attachments/610a4b259bced25d98e0df66/download/Register_a_Beacon_-_DAC_Accessibility_Audit_-_300721.pdf

Page 35

Reply:
changes made